connect/example/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/gandalf-network/gandalf-sdk-go/connect"
+)
+
+const publicKey = "0x036518f1c7a10fc77f835becc0aca9916c54505f771c82d87dd5943bb01ba5ca08";
+const redirectURL = "https://example.com"
+
+
+func main() {
+	qrCodeServices := connect.Services{
+		{Name: "Netflix", Status: true},
+		{Name: "Playstation", Status: false},
+	}
+	qrCodeURL, err := connect.GenerateQRCode(publicKey, redirectURL, qrCodeServices)
```

Comment:
This implementation is slightly different from what we have here: https://github.com/gandalf-network/connect


i believe it should look like this in go
``go
conn, err := connect.NewConnect(...)

conn.generateURL()
conn. generateQRCode()
conn. getDataKeyFromURL()

``